### PR TITLE
Feature / activate main template using Iguana-theme

### PR DESCRIPTION
**Pull request by sbrody**
Makes the first step towards removing Roots and Mustache and using Iguana-theme to control the template. 
The Roots wrapper function is deactivated. 

Iguana theme is activated and a new main.php template file created. (This is currently situated in the root directory - setting /templates as the theme directory is perhaps a subject for another PR).

Mustache variables are replaced with native WordPress functions, strings or php variables. A new main-content template file is introduced to prevent existing child themes overriding it (the gds-blogs-root theme has its own base.php file - potential knock on effects on child theme will need to be explored further) and trying to call deactivated Roots functions.  Redundant template files are deleted.

Testing: Check that pages are displayed correctly and as they were before changes.

### CODE REVIEW

File: layouts/main.php [hunk 1]
```diff
@@ -0,0 +1,80 @@
+<?php
+    $govukFrontendAssetPath = get_template_directory_uri() . '/build/node_modules/govuk-frontend/dist/govuk/assets/';
+?>
+<!doctype html>
+<html lang="en" class="govuk-template">
+	<head>
+		<meta charset="utf-8" />
+		<title><?php echo esc_html(get_the_title()); ?></title>
```
[the remark this answers is not in the document]
Comment (sbrody) [reply layouts/main.php:8]: Ok - sounds like a good plan

File: layouts/main.php [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+		<link rel="mask-icon" href="<?php echo esc_attr($govukFrontendAssetPath); ?>images/govuk-icon-mask.svg" color="#0b0c0c">
+		<link rel="apple-touch-icon" href="<?php echo esc_attr($govukFrontendAssetPath); ?>images/govuk-icon-180.png">
+		<link rel="manifest" href="<?php echo esc_attr($govukFrontendAssetPath); ?>manifest.json">
+
+		<?php wp_head(); ?>
+	</head>
+	<body <?php body_class('govuk-template__body'); ?>>
+   		<script>document.body.className += ' js-enabled' + ('noModule' in HTMLScriptElement.prototype ? ' govuk-frontend-supported' : '');</script>
+
+   		<a href="#content" class="govuk-skip-link" data-module="govuk-skip-link">{{ skipLinkMessage }}</a>
```
[the remark this answers is not in the document]
Comment (sbrody) [reply layouts/main.php:24]: Missed that one

**Comment by sbrody** [comment]
> It should be OK to remove `base.php`. This theme is not a child theme of the _Root_ theme, so no relationship to worry about there.

But, counterintuitively, I believe the Root theme is a child of this one!

**Comment by matthewpassmore** [comment]
> > It should be OK to remove `base.php`. This theme is not a child theme of the _Root_ theme, so no relationship to worry about there.
> 
> But, counterintuitively, I believe the Root theme is a child of this one!

Ah yes, hold on deletion for now!

**Comment by sbrody** [comment]
> > > It should be OK to remove `base.php`. This theme is not a child theme of the _Root_ theme, so no relationship to worry about there.
> > 
> > 
> > But, counterintuitively, I believe the Root theme is a child of this one!
> 
> Ah yes, hold on deletion for now!

Do you think the two theme need to be on the same repo? I don't know how we manage this otherwise? If I don't replace base with main-content.php it essentially breaks the child theme, which uses its own base.php to call various template paths using the Root function which I needed to deactivate in the parent theme

**Comment by matthewpassmore** [comment]
> Do you think the two theme need to be on the same repo?

No. We can manage this by making the necessary change in the child theme, and ensuring that change is included in the same Production Release as the bump in versioning of this theme. That way the deployed changes should be in sync. Do you see any issues with that approach?

**Comment by sbrody** [comment]
> > Do you think the two theme need to be on the same repo?
> 
> No. We can manage this by making the necessary change in the child theme, and ensuring that change is included in the same Production Release as the bump in versioning of this theme. That way the deployed changes should be in sync. Do you see any issues with that approach?

I think that it's possible to do it this way but it seems messy - I'm guessing it's because the gds-blogs parent theme is the only one that is publicly available?

**Review by matthewpassmore — APPROVED** [review]
All working as expected 👍

**Pull request merged**